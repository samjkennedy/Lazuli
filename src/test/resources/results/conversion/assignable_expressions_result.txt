10
3
0
1
4
9
16
25
36
49
64
81
100
5050
5151
5253
5356
5460
5565
5671
5778
5886
5995
0
6
12
18
24
30
36
42
48
54
60
66
72
78
84
90
96
0
2
4
6
8
10
12
14
16
18
20
22
24
26
28
30
32
34
36
38
40
42
44
46
48
50
52
54
56
58
60
62
64
66
68
70
72
74
76
78
80
82
84
86
88
90
92
94
96
98
