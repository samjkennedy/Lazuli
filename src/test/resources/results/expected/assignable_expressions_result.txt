10
3
0
1
4
9
16
25
36
49
64
81
100
5050
5151
5253
5356
5460
5565
5671
5778
5886
5995
